packages/checkbox/src/index.mts

```diff
@@ -96,9 +96,14 @@ export default createPrompt(
 
     const [active, setActive] = useState(bounds.first);
     const [showHelpTip, setShowHelpTip] = useState(true);
+    const [showValidationMessage, setShowValidationMessage] = useState(false);
 
     useKeypress((key) => {
       if (isEnterKey(key)) {
+        if (items.filter(isChecked).length === 0) {
+          setShowValidationMessage(true);
+          return;
```

Comment:
Nit: Let's instead a `if/else` block. Same idea as before, since we'll eventually add a second block to handle custom validation, it'll fit right in and won't skip future code that might be useful (unlikely, but you could see cleanup code done later in the function.)